lib/Dialect/TorchConversion/Transforms/Passes.cpp

```diff
@@ -101,6 +106,14 @@ void TorchConversion::createTorchBackendToLinalgOnTensorsBackendPipeline(
   pm.addPass(TorchConversion::createVerifyLinalgOnTensorsBackendContractPass());
 }
 
+void TorchConversion::createOnnxBackendToLinalgOnTensorsBackendPipeline(
+    OpPassManager &pm) {
+  pm.addNestedPass<func::FuncOp>(onnx_c::createTorchOnnxToTorchPass());
+  Torch::TorchLoweringPipelineOptions options;
+  Torch::createTorchFunctionToTorchBackendPipeline(pm, options);
```

Comment:
Is this entire pipeline needed? From skimming the ONNX Python importer, it seems that it has similar guarantees as the FX Python importer (value semantics, shapes, dtypes, etc). If that's the case, then we don't need to perform the entire `TorchFunctionToTorchBackendPipeline`. I think you just need the decompositions pass.